telegram/commands/gemini/gemini.go

```diff
@@ -0,0 +1,25 @@
+package gemini
+
+import (
+	"activity-tracker/gemini"
+	"activity-tracker/shared"
+	"errors"
+	"fmt"
+)
+
+var (
+	ErrNoPrompt = errors.New("En serio metiste un prompt vacío bobo hijueputa? Haz algo bien sub-humano 🙄")
+)
+
+func HandleGemini(bot *shared.Bot, chatID int64, userName, content string) error {
+	if content == "" {
+		return ErrNoPrompt
+	}
+	response, err := gemini.Gemini(content + "responde solo texto")
```

Comment:
add a comment here, its too hacky and maybe we will not understand what it does if we need to change this code in the future

Reply:
additionally, set the string as a global var

Reply:
solved